spec/factories/story_factory.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :story do
+    sequence(:title) { |n| "Life Of Pi#{n}" }
+    sequence(:body) { |n| "I suppose in the end,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.